content/src/fieldConfig/callout-defaults.json

```diff
@@ -3,6 +3,11 @@
     "informationalHeadingDefaultText": "Keep in mind:",
     "conditionalHeadingDefaultText": "Once completed, you will have:",
     "warningHeadingDefaultText": "Be advised:",
-    "noteHeadingDefaultText": "Note:"
+    "quickReferenceHeadingDefaultText": "Quick Reference Placeholder Header Text:",
```

Comment:
The default label in the cms is "title", it serves as a placeholder. The default label I set makes it clear it needs to change. Content can change this if they like, I will leave a note for them in the ticket.